backend/src/models/answer.js

```diff
@@ -10,3 +10,16 @@ exports.getAnswers = function (questionID, pageNumber) {
         )
     })
 }
+
+exports.postAnswer = function (questionID, {body}) {
+    new Promise((resolve, reject) => {
+        const columns = ['userID', 'questionID', 'body']
+        const placeholders = columns.map(_ => '?').join()
+        const query = `INSERT INTO answer (${columns}) VALUES (${placeholders})`
+        db.run(
+            query,
+            [0, questionID, body], // TODO user id is a placeholder obviously, but it can't be null so...
+            function (err) { err ? reject(err) : resolve(exports.getAnswers(questionID, 1)) }
```

Comment:
@DarkPurple141 Basically I don't get what's going on here (this code doesn't work).
I wanna return getAnswers after adding the new answer to the db

Reply:
Fixed with a missing return XD.

Reply:
Sidenote I feel like calling things with exports.xxxx is sort of annoying.

See the way I made changes in db.js. Dunno!